modules/tensor_mechanics/src/materials/czm/CZMMaterialBase.C

```diff
@@ -0,0 +1,78 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "Assembly.h"
+#include "CZMMaterialBase.h"
+#include "RotationMatrix.h"
+
+template <>
+InputParameters
+validParams<CZMMaterialBase>()
+{
+  InputParameters params = validParams<InterfaceMaterial>();
+
+  params.addParam<Real>("inner_penetration_penalty", 1, "inner penalty factor");
```

Comment:
Do you mean 'inter-penetration penalty'?
Actually, this parameter isn't even used.  Get rid of it.

Reply:
ok will put it back in models with damage